recipes/spades/3.12.0/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set version = "3.12.0" %}
```

Comment:
do we need to keep a copy of this? we can always pull it from git history if we need to, can't we?